docs/building-spark.md

```diff
@@ -70,6 +70,8 @@ This will build Spark distribution along with Python pip and R packages. For mor
 
 ## Specifying the Hadoop Version and Enabling YARN
 
+You can enable the exact profile of Hadoop to compile against through `-Phadoop-2.7`(default) and `-Phadoop-3.2`.
+
```

Comment:
Here, `exact profile` might look strange because we also have `hadoop-provided` profile. What about just omitting this line 73~74? The following example (82~84) looks clear enough to me (although it's an implicit way.)

Also, could you update the PR title because this PR includes `hadoop` example update additionally?